Sources/MessagingInApp/Gist/Gist.swift

```diff
@@ -12,6 +14,10 @@ public class Gist: GistDelegate {
 
     public static let shared = Gist()
 
+    init() {
+        self.logger = DIGraphShared.shared.logger
```

Comment:
why not just use ?

```
private let logger: Logger = DIGraphShared.shared.logger
```

Reply:
`Gist` class is moved to `DIGraphShared ` in #802, so `DIGraphShared` usage is replaced with constructor values. I kept it in the constructor to make the code changes a bit simpler in next PRs.